crates/uv-tool/src/lib.rs

```diff
@@ -175,11 +187,12 @@ impl InstalledTools {
     /// # Errors
     ///
     /// If no such environment exists for the tool.
-    pub fn remove_environment(&self, name: &PackageName) -> Result<(), Error> {
+    pub fn remove_environment(&self, name: &ToolName) -> Result<(), Error> {
         let environment_path = self.tool_dir(name);
 
         debug!(
-            "Deleting environment for tool `{name}` at {}",
+            "Deleting environment for tool `{}` at {}",
+            name.name,
```

Comment:
We'll generally probably want to see the both the name and suffix that we're performing operations on.

Reply:
I'm a bit curious why you did this instead of implementing `Display` for `ToolName`